internal/testhelpers/testhelpers.go

```diff
@@ -77,6 +78,9 @@ func AssertStateSentTo(t *testing.T, ses protocols.SideEffects, expected state.S
 
 func AssertProposalSent(t *testing.T, ses protocols.SideEffects, sp consensus_channel.SignedProposal, to testactors.Actor) {
 
+	if len(ses.MessagesToSend) != 1 {
```

Comment:
Clean this up

Reply:
[f0e30e6](https://github.com/statechannels/go-nitro/pull/826/commits/f0e30e60ac5c98547e2ee632e6eb688bf0cc44cc)